students/FlorentinPopescu/09_Kata_fourteen_L4_FP.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Jan 20 00:04:07 2019
+@author: Florentin Popescu
+"""
+
+#===================LESSON_04====================
+# Kata fourteen - Tom Swift under milk wood------
+#================================================
+
+#================================================
+# Trigrams
+#================================================
+
+#------------------------------------------------
+# Option 1 - construct a trigram
+#------------------------------------------------
+import re
+import random
+import string
+
+#------------------------------------------------
+def trigram(words):
+    trigram = [(words[i], words[i+1], words[i+2]) for i in range(len(words) - 2)]
+    return trigram
+
+#------------------------------------------------
+def generate_text(trigram, number_sentences = 10, sentence_lenght= 5):
+    trigram_dict = {trigram[i][0:2]:[trigram[i][2]] for i in range(1, len(trigram))}
+    
+    """ generate text """
+    text = ''
+    for i in range(number_sentences):  
+        # start with a random pair of neighbors
+        chain = random.choice(list(trigram_dict.keys()))
+        sentence = list(chain) 
+    
+        for j in range(sentence_lenght):
+            #generate new key from old key and new word
+            try: 
+                chain = (chain[1], trigram_dict.get(chain)[0])
+                sentence.append(chain[1])
+            except:
+                break
+
+        sentence[0] = sentence[0].capitalize()
+        sentence[-1] += ". "
+        sentence = " ".join(sentence)
+        text += sentence
+    
+    return text
+
+#------------------------------------------------
+def main():
+    #Open the file for reading 
+    try:
+        with open("sherlock_small.txt", 'r') as f:
+            small_text = f.read()
+    except: FileNotFoundError("File indicated not found.")

```

Comment:
are you attempting to raise an exception here? also, should be two separate lines